src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -2,12 +2,8 @@
   position: relative;
 }
 
-/**
- * 1. Need to override the react grid layout height when a single panel is expanded. Important is required because
- * otherwise the height is set inline.
- */
```

Comment:
I missed this `react-grid-layout`-specific comment in the original replacement - oops 🤦